src/freenet/node/PeerManager.java

```diff
@@ -1785,8 +1785,8 @@ public void maybeLogPeerNodeStatusSummary(long now) {
 	
 	public void changePeerNodeStatus(PeerNode peerNode, int oldPeerNodeStatus,
 			int peerNodeStatus, boolean noLog) {
-		Integer newStatus = Integer.valueOf(peerNodeStatus);
-		Integer oldStatus = Integer.valueOf(oldPeerNodeStatus);
+		Integer newStatus = peerNodeStatus;
+		Integer oldStatus = oldPeerNodeStatus;
```

Comment:
Okay, okay, hold up. Now these two lines are completely stupid and need to be removed. That’s just renaming! 🙂